### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by stellakunzang**
Readme is complete!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,8 +20,8 @@ Once we verified that all of our methods were performing as expected in all clas
...
+The Gameable, Teamable, and Gameteamable modules were each created to reduce repetition in their respective classes. We were running very similar code within many methods and found that by pulling the methods out into a module and passing in an argument we were able to reduce repetition drastically. We also decided to move some of our helper methods into these class-specific modules. Often methods were created in pairs (most/least, worst/best, etc.) and the same helper method would be called by both methods in the pair. In these cases, we moved the helper method into a module, along with any helper methods that the helper method relied on. This reorganization has made our classes more streamlined; they now primarily contain methods whose output is valuable to us in terms of statistical analysis and which are integral to the functioning of the centralized StatTracker class.
```
Comment (perryr16) [inline README.md:25]: This is a thorough explanation. 100%

File: README.md [hunk 2]
```diff
@@ -17,8 +20,8 @@ Once we verified that all of our methods were performing as expected in all clas
...
 
-Our modules are called Gameable, Hashable, and Winable. Hashable was extremely useful and showed us all the value of using hashes, and moreover a hash within a hash, to reduce the need for nested iteration. It allowed us to, in a very direct way, iterate over our data to calculate the desired values, and then to divide those values to obtain a percentage. What had originally taken many lines and helper methods was reduced exponentially by implementing the Hashable module and extensively refactoring to utilize the methods contained therein.
+Hashable is our most versatile module and has been utilized extensively by multiple classes. It was the first one we created and it showed us the value of using hashes, and moreover a hash within a hash, to reduce the need for nested iteration. It allowed us to, in a concise and direct way, iterate over our data to calculate the desired values, and then to divide those values to obtain a percentage. What had originally taken many lines and helper methods was reduced exponentially by implementing the Hashable module and refactoring to utilize the methods contained therein.
```
Comment (Rostammahabadi) [inline README.md:23]: giphy (1)

**Review by perryr16 — COMMENTED** [review]
approved -rp

File: README.md [hunk 3]
```diff
@@ -17,8 +20,8 @@ Once we verified that all of our methods were performing as expected in all clas
...
 
-Overall, our design philosophy was governed by a collective desire to reduce unnecessary repetition and excessive nested iteration, and produce elegant, clean, concise Ruby code. We found that inheritance and modules were powerful new tools to accomplish this goal. 
+Overall, our design philosophy was governed by a collective desire to reduce unnecessary repetition and excessive nested iteration, and produce elegant, clean, and concise Ruby code. We found that inheritance and modules were powerful new tools to accomplish this goal.
```
Comment (Rostammahabadi) [inline README.md:27]: This looks great

**Pull request merged**